tests/test_mdlp.py

```diff
@@ -32,165 +31,46 @@ def test_find_cut_no_cut():
     k = find_cut(y, 0, len(y))
     assert_equal(-1, k)
 
-def test_mdlp_iris():
-    from sklearn.datasets import load_iris
+def test_fit_transform():
+  expected = [
+    [0, 0],
+    [0, 0],
+    [1, 0],
+    [2, 0],
+  ]
+
+  X = np.array([
+    [0.1, 0.1],
+    [0.2, 0.4],
+    [0.3, 0.2],
+    [0.4, 0.3]
+  ])
+  y = np.array([0, 0, 1, 2])
+  for i in range(10):
+    scaled_disc = MDLP(shuffle=False).fit_transform(X / 10**i, y)
+    assert_array_equal(expected, scaled_disc)
+
+def test_translation():
+  expected = np.array([0, 0, 0, 0, 1, 1, 1, 1, 1]).reshape(-1, 1)
+
+  X = np.arange(9, dtype=float).reshape(-1, 1)
+  y = np.array([0, 0, 0, 0, 1, 0, 1, 1, 1])
+  transformed = MDLP(shuffle=False).fit_transform(X, y)
+  assert_array_equal(expected, transformed)
+
+  # translating data does not affect discretization result
+  translated = MDLP(shuffle=False).fit_transform(X - 5, y)
+  assert_array_equal(expected, translated)
 
-    iris = load_iris()
-    X = iris.data
-    y = iris.target
-    mdlp = MDLP(shuffle=False)
-    transformed = mdlp.fit_transform(X, y)
+def test_coerce_list():
```

Comment:
I appreciate the thought of adding this test, because I previously did not test whether we can use numpy arrays versus lists.

Thanks. If this were the standard scikit-learn library, there would already be unit tests for this functionality.